timesketch/frontend/src/views/SigmaContent.vue

```diff
@@ -0,0 +1,51 @@
+<!--
+Copyright 2020 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <div>
+        <section class="section">
+
+    <ts-navbar-main>
+      <template v-slot:left>
+        {{ sketch.name }}
+      </template>
+    </ts-navbar-main>
+
+    <ts-navbar-secondary currentAppContext="sketch" currentPage="sigma"></ts-navbar-secondary>
+      <div class="container is-fluid">
```

Comment:
weird indentation here. I'll show how to set up auto formatting that will help here :)
Install `prettier` in vscode, and config it to auto format on save:
https://scottsauber.com/2017/06/10/prettier-format-on-save-never-worry-about-formatting-javascript-again/

Reply:
TIL, thx!